index.html

```diff
@@ -6,20 +6,31 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome Books</title>
-  <link rel="stylesheet" href="reset.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
   <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
   <main>
-    <h1>Awesome Books</h1>
-    <div class="books-wrapper">
+    <div class="container-md d-flex flex-column align-items-center">
+      <h1 class="h1 text-center my-4">All Awesome Books</h1>
+      <table class="table table-striped">
+        <tbody class="books-wrapper">
+        </tbody>
+      </table>
+      <hr class="my-6">
+      <form class="card px-4 w-50">
+        <div class="card-body">
+          <h4 class="card-title text-center mb-3">Add a new Book</h4>
+          <input type="text" class="form-control my-2" placeholder="Title" id="new-book-title">
+          <input type="text" class="form-control my-2" placeholder="Author" id="new-book-author">
+          <div class="text-end">
+            <button type="button" class="add-btn btn btn-primary mt-3"> Add </button>
+          </div>
+        </div>
+      </form>
     </div>
```

Comment:
Make sure that this section is align-center according to the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). And make sure that the book title, writer name and remove button should be in the same row.